### DISCUSSION TITLE
Gitui Zed extension

### DISCUSSION THREAD

**Discussion by maljukan** [extrawurst/gitui#2224] [Ideas]
I wonder if anybody thought about this.
GitUI Extension for Zed in GPUI would be amazing.
GitUI would benefit from it as Zed users would get familiar with it and many of them would use GitUI when they're not using Zed.

**Comment by extrawurst** [comment]
Doesn’t it already just work inside zeds terminal? What would an extension do?

**Reply by maljukan** [reply]
Oh i was just never a fan of integrated terminals but now i opened it and... 
Khm khm this conversation never happened!
